src/component/handlers/composition/DraftEditorCompositionHandler.js

```diff
@@ -183,16 +249,18 @@ const DraftEditorCompositionHandler = {
         EditorState.push(editorState, contentState, 'insert-characters'),
       );
       return;
+      */
     }
-
+    onInput(editor);
```

Comment:
Is there any reason we _shouldn't_ just call the same function that handles spellchecks? This seems to make android just *work*.

Reply:
Also i'm not sure how to test this because the jest test seem to have document.getSelection and global.getSelection = undefined. Not sure how to emulate this.